show_page.html

```diff
@@ -44,6 +44,7 @@
 					<div class="show">
 						<img src="http://placehold.it/250x150">
 			      <h2>Radiolab</h2>
+						<div class="social"><img src="http://cdn.business2community.com/wp-content/uploads/2015/07/Twitter-icon.png.png"><img src="https://worldvectorlogo.com/logos/facebook-icon-white.svg"><img src="https://camo.githubusercontent.com/b13830f5a9baecd3d83ef5cae4d5107d25cdbfbe/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f3732313033382f313732383830352f35336532613364382d363262352d313165332d383964312d3934376632373062646430332e706e67"></div>
```

Comment:
Be careful when hotlinking files. Best thing to do, is download them and reference them yourself. If they change the url or even find out you are using their content, you may find yourself with a broken site/image.